src/routes/commentsRoutes.ts

```diff
@@ -4,18 +4,201 @@ import {authMiddleware} from '../controllers/usersController';
 
 const router = Router();
 
+/**
+ * @swagger
+ * tags:
+ *   name: Comments
+ *   description: The Comments API
+ */
+
+/**
+ * @swagger
+ * components:
+ *   schemas:
+ *     Comment:
+ *       type: object
+ *       required:
+ *         - postId
+ *         - userId
+ *         - content
+ *       properties:
+ *         _id:
+ *           type: string
+ *           description: The auto-generated id of the comment
+ *         postId:
+ *           type: string
+ *           description: The id of the post that the comment belongs to
+ *         userId:
```

Comment:
UserId is not required in all the routes. We decode it from the given jwt

Reply:
removed